src/gridster.component.ts

```diff
@@ -79,7 +80,7 @@ export class GridsterComponent implements OnInit, AfterContentInit, OnDestroy {
 
 
     gridsterOptions: GridsterOptions;
-    private subscribtions: Array<Subscription> = [];
+    private subscription: Subscription;
```

Comment:
`private subscription: Subscription = new Subscription();`
I would prefer to init Subscription directly and later always add, to always use `subscription.add(...)` without thinking whether it's first or not.
In general I like idea with Subscription best.

Reply:
Okay, PR updated